api/src/main/java/org/eclipse/microprofile/reactive/streams/ConnectingOperators.java

```diff
@@ -33,32 +33,32 @@
     /**
      * Connect the outlet of the {@link Publisher} built by this builder to the given {@link Subscriber}.
      *
-     * @param subscriber The subscriber to connect.
+     * @param subscriber The subscriber to connect, it should not be reused. 
      * @return A completion builder that completes when the stream completes.
      */
     ProducesResult<Void> to(Subscriber<? super T> subscriber);
 
     /**
      * Connect the outlet of this stream to the given {@link SubscriberBuilder} graph.
      *
-     * @param subscriber The subscriber builder to connect.
+     * @param subscriberBuilder The subscriber builder to connect.
      * @return A completion builder that completes when the stream completes.
      */
-    <R> ProducesResult<R> to(SubscriberBuilder<? super T, ? extends R> subscriber);
+    <R> ProducesResult<R> to(SubscriberBuilder<? super T, ? extends R> subscriberBuilder );
 
     /**
      * Connect the outlet of the {@link Publisher} built by this builder to the given {@link ProcessorBuilder}.
      *
-     * @param processor The processor to connect.
-     * @return A stream builder that represents the passed in processors outlet.
+     * @param processorBuilder The processor builder to connect.
+     * @return A stream builder that represents the passed in processor's outlet.
      */
-    <R> ConnectingOperators<R> via(ProcessorBuilder<? super T, ? extends R> processor);
+    <R> ConnectingOperators<R> via(ProcessorBuilder<? super T, ? extends R> processorBuilder );
```

Comment:
As with the comment on `to`.

Reply:
tried to improve the text in the latest commit.